.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,39 @@
+name: PR Build
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: Build apps
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        app-platform: [ ruby-on-rails ]
+        instrumentation-type: [ manual ]
+    env:
+      APP_PATH: sample-apps/${{ matrix.instrumentation-type}}-instrumentation/${{ matrix.app-platform }}
+    steps:
+      - name: Checkout This Repo
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Construct Sample App image tag
+        run: >
+          echo "APP_IMAGE=public.ecr.aws/aws-otel-test/ruby-${{ matrix.app-platform }}-${{ matrix.instrumentation-type }}:${{ github.sha }}" |
+          tee --append $GITHUB_ENV;
+      - name: Build Docker image
+        uses: docker/build-push-action@v2
```

Comment:
Key point is no `push: true`

Reply:
I think it makes perfect sense. since we don't want to push the images during the `pr-build` workflow ?

Reply:
Yup